packages/typo3-fractor/src/Helper/TYPO3StringUtility.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+declare(strict_types=1);
+
+namespace a9f\Typo3Fractor\Helper;
+
+use Symfony\Component\String\UnicodeString;
+
+class TYPO3StringUtility
+{
+    public static function prepareExtensionName(string $extensionName, int $delimiterPosition): string
```

Comment:
Is this method used anywhere? If not, we should remove it and remove also the dependency from symfony/string

Reply:
Not atm but there are rules that need this. The logic can be simplified as the core also doesn't use symfony. Shall I add it when I add the rule later on or now?